### PR TITLE
dns/ddclient Added OVHCLoud

### PR DESCRIPTION
**Pull request by MrRoundRobin**
Added ovh for ddclient according to this:
https://github.com/ddclient/ddclient/blob/develop/ddclient.in#L6339-L6356

As I have no account at OVH i cloudn't test it.

Solves #2775

### CODE REVIEW

File: dns/ddclient/src/opnsense/service/templates/OPNsense/ddclient/ddclient.conf [hunk 1]
```diff
@@ -54,6 +54,8 @@ server=ipv4.nsupdate.info
 protocol=dyndns2
 ssl=yes
 server=ipv6.nsupdate.info
+{%            elif account.service == 'ovh' %}
```
Comment (AdSchellevis) [inline dns/ddclient/src/opnsense/service/templates/OPNsense/ddclient/ddclient.conf:57]: you can omit the client config if it's only the protocol due to :
https://github.com/opnsense/plugins/blob/7fe1eaa4fcf74ad0fb5b3311eb04a45819e4fd78/dns/ddclient/src/opnsense/service/templates/OPNsense/ddclient/ddclient.conf#L64-L65

Comment (MrRoundRobin) [reply dns/ddclient/src/opnsense/service/templates/OPNsense/ddclient/ddclient.conf]: Ah right, missed that 👍🏼

**Comment by AdSchellevis** [comment]
Someone with ovh might want to try this first, looking at the ddclient source code I don't think `ovh` is defined as a protocol/service (but I could be mistaken)

https://github.com/ddclient/ddclient/blob/v3.9.1/ddclient#L497-L738

**Comment by MrRoundRobin** [comment]
🤦🏼‍♂️ I looked that up in the develop branch. Seems like it will be added in version 3.10.0

I'll convert this PR to draft until the updated package is available.

**Comment by AdSchellevis** [comment]
hehe, I had some similar issues when writing the plugin too, master/development is quite different in supported options.

**Comment by toxic0berliner** [comment]
Meanwhile I've included to my other edits the addition of OVH as a standard dyndns2 protocol with url www.ovh.com

That works fine for me at home and avoids the need to create a custom and specify the url each time. 
Really not sure what this protocol ovh adds on top of dyndns2 besides hardcoding the url of ovh.

If my #2883 goes through first this one should not clash anyway I chose another name ;)

**Review by satrapes — COMMENTED** [review]
I think that you are missing a change to root/dns/ddclient/src/opnsense/service/templates/OPNsense/ddclient/ddclient.conf

**Comment by AdSchellevis** [comment]
should be in the development version now.

**Pull request closed without merging**